### PR TITLE
Fix error in comments for LSTMLayer and GRULayer

### LINKED ISSUE #724: Error in LSTMLayer comment
Hello everyone!

I was reading through the code for the LSTM layer (and trying to implement LN-LSTM) that I came across this line in the code comments [recurrent.py#L990](https://github.com/Lasagne/Lasagne/blob/master/lasagne/layers/recurrent.py#L990).
Shouldn't at each call to scan, `input_n` be of size `(n_batch, 4*num_units)`. I know it is a minor issue, but just for the sake of correctness.

### PR DESCRIPTION
**Pull request by erfannoury**
Fixes #724

### CODE REVIEW

File: lasagne/layers/recurrent.py [hunk 1]
```diff
@@ -1371,7 +1371,7 @@ def get_output_for(self, inputs, **kwargs):
             # input is then (n_batch, n_time_steps, 3*num_units).
             input = T.dot(input, W_in_stacked) + b_stacked
 
-        # At each call to scan, input_n will be (n_time_steps, 3*num_units).
+        # At call to step from theano.scan, input_n will be (n_batch, 4*num_units).
```
Comment (f0k) [inline lasagne/layers/recurrent.py]: 3 != 4

File: lasagne/layers/recurrent.py [hunk 2]
```diff
@@ -987,7 +987,7 @@ def get_output_for(self, inputs, **kwargs):
             # (n_time_steps, n_batch, 4*num_units).
             input = T.dot(input, W_in_stacked) + b_stacked
 
-        # At each call to scan, input_n will be (n_time_steps, 4*num_units).
+        # At call to step from theano.scan, input_n will be (n_batch, 4*num_units).
```
Comment (f0k) [inline lasagne/layers/recurrent.py]: pep8: line too long

Maybe start with `When theano.scan calls step,`

**Comment by erfannoury** [comment]
@f0k I'm sorry I acted too carelessly 🙍 
I think it's fixed now.

**Comment by f0k** [comment]
Thanks, looks good now. Merging!

**Pull request merged**